### PR TITLE
Update icons xml

### PR DESCRIPTION
**Pull request by jensens**
- update icons.xml (icons were already up to date thanks to package json)
- update iconsmap.json
- add update steps to Makefile for future automatation.

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by thet — CHANGES_REQUESTED** [review]
LGTM except the requested change.
Are there any new icons to be installed in ``static/icons-bootstrap``?
If so, they would need to get added in a separate commit.
But IIRC these icons were added in a previous commit, right?

File: Makefile [hunk 1]
```diff
@@ -9,8 +9,12 @@ clean:
...
 
+.PHONY: update-icons
+update-icons:
+	(cd src/plone/staticresources/_scripts && `which python3` register_icons.py)
+	(cd src/plone/staticresources/_scripts && `which python3` register_flag_icons.py)
+	(cd src/plone/staticresources/_scripts && `which python3` iconmap_json.py)
+
 .PHONY: build
-build:: clean install
+build:: clean install update-icons
```
[the remark this answers is not in the document]
Comment (jensens) [reply Makefile]: Fixed. Makefile has now a sane dependency chain and an "all" step as first.

**Comment by jensens** [comment]
> Are there any new icons to be installed in `static/icons-bootstrap`?

No. The existing ones were not registered completely.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by thet — APPROVED** [review]

**Pull request merged**